### PR TITLE
Submit ZeroOne BootLoader and Readme File.

### PR DESCRIPTION
**Pull request by ZeroOne-Aero**
Submit hwdef.dat hwdef-bl.dat defaults.parm and readme file.

### CODE REVIEW

**Comment by ZeroOne-Aero** [comment]
Also submitted ZeroOneX6_bl.bin and ZeroOneX6_bl.hex.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+define HAL_GPIO_PWM_VOLT_PIN 3
+define HAL_GPIO_PWM_VOLT_3v3 0
+
+# we need to disable DMA on the last 2 FMU channels
+# as timer 12 doesn't have a TIMn_UP DMA option
+PH6  TIM12_CH1 TIM12 PWM(7) GPIO(56) NODMA
+PH9  TIM12_CH2 TIM12 PWM(8) GPIO(57) NODMA
+
+# GPIOs
+PE11 FMU_CAP1 INPUT GPIO(58)
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:177]: which connector/pin name?

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+define HAL_HEATER_GPIO_PIN 80
+
+# Setup the IMU heater
+define HAL_HAVE_IMU_HEATER 1
+define HAL_IMU_TEMP_DEFAULT 45
+define HAL_IMUHEAT_P_DEFAULT 50
+define HAL_IMUHEAT_I_DEFAULT 0.07
+
+# armed indication
+PE6 nARMED OUTPUT HIGH
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:221]: where is this pinned out/connector?

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm [hunk 3]
```diff
@@ -0,0 +1,8 @@
+BATT_VOLT_PIN 12
```
[the remark this answers is not in the document]
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm]: I have deleted these two lines:
BATT_VOLT_PIN 12
BATT_CURR_PIN 13

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+# ADC
+PA0 SCALED1_V3V3 ADC1 SCALE(2)
+PA4 SCALED2_V3V3 ADC1 SCALE(2)
+PB0 SCALED3_V3V3 ADC1 SCALE(2)
+PF12 SCALED4_V3V3 ADC1 SCALE(2)
+
+PB1 VDD_5V_SENS  ADC1 SCALE(2)
+
+# pin7 on AD&IO, analog 12
+PC2 ADC1_6V6 ADC1 SCALE(2)
```
[the remark this answers is not in the document]
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:111]: PC2
PC2 is the ADC 6V6 label on the AD&IO terminal, which I marked in this diagram.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+# pin6 on AD&IO, analog 13
```
[the remark this answers is not in the document]
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:113]: PC3
PC3 is the ADC 3V3 label on the AD&IO terminal, which I marked in this figure.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat (hunk 1 above)
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:177]: PH9 PE11
PE11 is the CAP1 label on the AD&IO terminal, PH9 is the M16 label on the M15M16 terminal, which I have marked in this figure.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+
+# ADC
+PA0 SCALED1_V3V3 ADC1 SCALE(2)
+PA4 SCALED2_V3V3 ADC1 SCALE(2)
+PB0 SCALED3_V3V3 ADC1 SCALE(2)
+PF12 SCALED4_V3V3 ADC1 SCALE(2)
+
+PB1 VDD_5V_SENS  ADC1 SCALE(2)
+
+# pin7 on AD&IO, analog 12
```
[the remark this answers is not in the document]
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:110]: PB1 stands for onboard power supply voltage monitoring and is not a connector.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat (hunk 2 above)
Comment (ZeroOne-Aero) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat:221]: PE6
PE6 is the nARMED label on the AD&IO terminal, which I marked in this picture.

File: libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/defaults.parm]: tnx

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**